go.mod

```diff
@@ -5,10 +5,13 @@ go 1.24
 toolchain go1.24.0
 
 require (
+	github.com/ethereum/go-ethereum v1.15.3
 	github.com/hashicorp/go-plugin v1.6.3
 	github.com/leanovate/gopter v0.2.11
+	github.com/pkg/errors v0.9.1
 	github.com/shopspring/decimal v1.4.0
-	github.com/smartcontractkit/chainlink-common v0.4.1
+	github.com/smartcontractkit/chainlink-common v0.4.2-0.20250130202959-6f1f48342e36
+	github.com/smartcontractkit/chainlink-integrations/evm v0.0.0-20250317165050-f8c18e1f415e
```

Comment:
Hmm why are the chain specifics showing up in this module? Does the evm codec run in the ReportingPlugin rather than the EVM Relayer?

Reply:
Oh well is this module only imported for `github.com/smartcontractkit/chainlink-integrations/evm/utils/big`? Can we just copy a few util types?

Reply:
Yes the EVM codec runs in the plugin.

And it looks like its only copied for that big type as you mentioned, I suspect we can just copy it

Reply:
Done